pkg/kubelet/kubelet.go

```diff
@@ -1078,6 +1089,14 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 	return node, nil
 }
 
+func (kl *Kubelet) providerRequiresNetworkingConfiguration() bool {
+	if kl.cloud == nil || kl.flannelExperimentalOverlay {
+		return false
+	}
+	// TODO: Should we include any other providers here?
+	return kl.cloud.ProviderName() == gce.ProviderName
```

Comment:
This belongs in the cloudprovider package. We don't check provider names outside that package.